### ISSUE TITLE
Add LDAP support.

### ISSUE THREAD

**Issue by Denney-tech** [ansible/galaxy-operator#96]
**Is your feature request related to a problem? Please describe.**
Please add ldap support. The `spec.ldap.config` CRD from the old pulp resource type is completely ignored in the new operator.  In theory, all of the requisite values can be added in `pulp_settings:`, but there's some problems with values being treated as strings instead of dictionaries or methods.

E.g. `AUTH_LDAP_GROUP_TYPE = "NestedActiveDirectoryGroupType()"` should be `AUTH_LDAP_GROUP_TYPE = NestedActiveDirectoryGroupType()`

**Describe the solution you'd like**
I like the way `ldap_password_secret:` and `extra_settings:` work in AWX-Operator better than in the Pulp-Operator. I've had better luck with setting all the right values there.

**Describe alternatives you've considered**
Pulp-operator "works" for me, but just barely.

**Additional context**
I just want Galaxy to play nicely with AD.

**Comment by Denney-tech** [comment]
#85 fixes `AUTH_LDAP_GROUP_TYPE = NestedActiveDirectoryGroupType()` specifically, but it would still be preferred to have an ldap secret to apply the ldap password to.

**Comment by josemgom** [comment]
Agree. It would be better if we have a way to define the password in a secret, that would open the door to use something like sealed secrets to store it in git repositories.
